process-migration-service/src/test/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    version="2.1"
+    xmlns="http://xmlns.jcp.org/xml/ns/persistence"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
+    <persistence-unit name="migration-test" transaction-type="RESOURCE_LOCAL">
+        <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+        <class>org.kie.processmigration.model.Plan</class>
+        <class>org.kie.processmigration.model.Migration</class>
+        <class>org.kie.processmigration.model.MigrationReport</class>
+        <class>org.kie.processmigration.model.Credentials</class>
```

Comment:
why Credentials is here and not in the main persistence.xml? See 
https://github.com/kiegroup/droolsjbpm-integration/pull/1575/files#diff-c414854ad24e6abc450694bc39c11fc8

Reply:
Legacy from old implementation. Removing it.